src/aksharamukha/index.ts

```diff
@@ -40,6 +40,9 @@ export type AksharamukhaInitOptions = {
 export default class Aksharamukha {
 	static _isTestEnv: boolean = false;
 	static _currentScript: HTMLScriptElement;
+	static _pyodidePromise: Promise<PyodideInterface> | null = null;
```

Comment:
This implementation is reusing the same Pyodide instance across all instances of Aksharamukha. That's not correct – we want each instance of the Aksharamukha class to have its own unique Pyodide instance.

Maybe what we could do instead is have an async function that would assign a promise to something like `_nextPyodidePromise` at the time of importing the library, and then at the time of instantiation (calling `.new(...)`), await that instance and use that for creating the client, while assigning a fresh new initialization promise to `_nextPyodidePromise` so that it can instantiate in the background.